src/lib/utils.js

```diff
@@ -0,0 +1,18 @@
+import { debounce, throttle } from 'lodash';
+
+export const isFunction = fn => typeof fn === 'function';
```

Comment:
Let's use `lodash`'s `isFunction` method https://lodash.com/docs/4.17.11#isFunction

Reply:
I use native function to light and fast than lodash fn